pkgs/by-name/st/star/package.nix

```diff
@@ -20,31 +24,48 @@ stdenv.mkDerivation rec {
   sourceRoot = "${src.name}/source";
 
   postPatch = ''
-    substituteInPlace Makefile --replace "/bin/rm" "rm"
+    substituteInPlace Makefile --replace-fail "-std=c++11" "-std=c++14"
   '';
 
   nativeBuildInputs = [ xxd ];
 
-  buildInputs = [ zlib ];
+  buildInputs = [ zlib ] ++ lib.optionals stdenv.isDarwin [ llvmPackages.openmp ];
+
+  enableParallelBuilding = true;
+
+  makeFlags = lib.optionals stdenv.hostPlatform.isAarch64 [ "CXXFLAGS_SIMD=" ];
+
+  preBuild = lib.optionalString stdenv.isDarwin ''
+    export CXXFLAGS="$CXXFLAGS -DSHM_NORESERVE=0"
+  '';
 
   buildFlags = [
     "STAR"
     "STARlong"
   ];
 
-  enableParallelBuilding = true;
-
   installPhase = ''
     runHook preInstall
     install -D STAR STARlong -t $out/bin
     runHook postInstall
   '';
 
+  passthru.tests.version = testers.testVersion {
+    package = star;
+    command = "STAR --version";
+  };
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
https://github.com/NixOS/nixpkgs/blob/a2ff75328be8383feadb8a894b73f770f028531f/pkgs/README.md?plain=1#L867

Shall we create another issue to rename the package to `rna-star` to match repology?